src/UIAutomation/FunctionalTests/UiaOperationAbstractionTests.cpp

```diff
@@ -1934,5 +1934,42 @@ namespace UiaOperationAbstractionTests
         {
             UiaVariantCastUiaEnumTest(true /* useRemoteOperations */);
         }
+
+        // Asserts that you can convert a UiaElement to / from a UiaVariant 
+        void ElementAsVariantTest(const bool useRemoteOperations)
+        {
+            ModernApp app(L"Microsoft.WindowsCalculator_8wekyb3d8bbwe!App");
+            app.Activate();
+            auto calc = WaitForElementFocus(L"Display is 0");
+
+            auto guard = InitializeUiaOperationAbstraction(useRemoteOperations);
+
+            auto scope = UiaOperationScope::StartNew();
+
+            UiaElement element = calc;
+            auto ev = UiaVariant(element);
+            auto ev_IsElement = ev.IsElement();
```

Comment:
Just to avoid using underscores in names, I think either `elementVariantIsElement` or just `isElement` are both fine here. Same for `ev_AsElement` below.